spack/packages/fiber/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+FFT_LIBS = (
+    'heffte',
+    'fftw',
+    'ffte',
+)
+
+
+class Fiber(CMakePackage):
+    """High Performance Fast Fourier Transform benchmark."""
+
+    homepage = "https://fiber.icl.utk.edu/"
+    git      = "https://github.com/icl-utk-edu/fiber"
+
+    maintainers = ['G-Ragghianti', 'luszczek']
+
+    version('master', branch='master')
+
+    variant('fft', default='none', values=FFT_LIBS,
+            multi=True, description='Supported FFT libraries')
+    #variant('onlycomplex', default=False)
+
+    depends_on('mpi')
+
+    # Iterate over the fftlib variant for simple dependencies
+    for fft in FFT_LIBS:
+        depends_on(fft, when='fft={0}'.format(fft))
+
+    depends_on('heffte+fftw+cuda', when='fft=heffte')
+
+    def cmake_args(self):
+        args = []
+        for fft in self.spec.variants['fft'].value:
+            args.extend(["-DFIBER_ENABLE_{0}=ON".format(fft.upper())])
+            # These don't seem to be required within spack 
+            #args.extend("-DFIBER_FFT_INCLUDE_DIRS={0}".format(self.spec[fft].prefix.include))
+            #args.extend("-DFIBER_FFT_LIB_DIRS={0}".format(self.spec[fft].prefix.lib))
+        return args
+
+    def install(self, spec, prefix):
```

Comment:
Install should go into CMake. Leaving it here will override whatever CMake has in the future. That's what #10 is about.

Reply:
Yes, I just needed this part in order for the spack install to work right now.  Once the "make install" is in cmake, I will be able to remove this.